### PR TITLE
Fix video popup on windows

### PR DESCRIPTION
**Pull request by patrickelectric**
This is a weird issue, and very inconsistent, this patch **appears** (we are talking about windows, nothing is for sure) to solve the problem.
Besides me, someone can test and confirm ?
Also, replace #6365.
[AppVeyor download link.](https://ci.appveyor.com/api/buildjobs/rsrnqcvg8pk71q3e/artifacts/QGroundControl-installer.exe)

### CODE REVIEW

File: src/VideoStreaming/VideoReceiver.cc [hunk 1]
```diff
@@ -872,7 +874,7 @@ VideoReceiver::_updateTimer()
                 stop();
             }
         } else {
-            if(!running() && !_uri.isEmpty() && _videoSettings->streamEnabled()->rawValue().toBool()) {
+            if(!_stop && !running() && !_uri.isEmpty() && _videoSettings->streamEnabled()->rawValue().toBool()) {
```
Comment (jaxxzer) [inline src/VideoStreaming/VideoReceiver.cc]: We should call this `_stopped`

Comment (patrickelectric) [reply src/VideoStreaming/VideoReceiver.cc]: Theoretically, this variable comes from the user input and not to handle the pipeline, if stop comes after "stopping the video stream" I would agree.

**Comment by dogmaphobic** [comment]
The code "looks" right. I won't be able to test this (Windows) any time soon though...

**Comment by patrickelectric** [comment]
This PR appears to solve ubuntu problem also, as described [here](https://github.com/bluerobotics/qgroundcontrol/issues/164#issuecomment-383749346).

**Comment by patrickelectric** [comment]
It appears to have a problem related to this modifications, I'll put it as WIP just to make sure.

**Comment by patrickelectric** [comment]
Correction done.

**Comment by DonLakeFlyer** [comment]
@patrickelectric Good to merge now?

**Comment by patrickelectric** [comment]
Yep, everything was tested and working.

**Pull request merged**